openpype/hosts/maya/api/plugin.py

```diff
@@ -300,6 +300,37 @@ def update(self, container, representation):
                      str(representation["_id"]),
                      type="string")
 
+        # Update any xgen containers.
+        compound_name = "xgenContainers"
+        object = "SplinePrimitive"
```

Comment:
It seems the `object` "SplinePrimitive" isn't used until the `xgenm.setAttr` call. How about just putting that string directly into the function call?